generic/tests/netperf.py

```diff
@@ -130,6 +130,13 @@ def mtu_set(mtu):
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
+        if params.get("netdst_nic1") in process.system_output(
+                "ovs-vsctl list-br", ignore_status=True, shell=True).decode():
```

Comment:
I think no need "ignore_status=True" here. It should raise error.

Reply:
@YanhuiMa Should consider no ovs environment. Thanks.

Reply:
@YanhuiMa I am not sure if it works well on both environments, so beg you help test it. Thanks.

Reply:
I have test it before, it works in ovs/i440fx environments

Reply:
Yes, I will fix it.